op-acceptance-tests/tests/isthmus/operator_fee/test_case_generator.go

```diff
@@ -0,0 +1,147 @@
+package isthmus
+
+import (
+	cryptoRand "crypto/rand"
+	"fmt"
+	"math"
+	"math/big"
+	"math/rand"
+	"time"
+)
+
+// init seeds the global PRNG using the current time.
+var seededRand *rand.Rand // Added package-level generator
+
+func init() {
+	// rand.Seed(time.Now().UTC().UnixNano()) // Deprecated
+	seededRand = rand.New(rand.NewSource(time.Now().UTC().UnixNano())) // Use new generator
```

Comment:
I see that we need this kind of seeding for fuzzing, it would be useful for logging the time for seeding for reproduction.